### ISSUE TITLE
need heteroscedastic example and/or tutorial

### ISSUE THREAD

**Issue by bwpriest** [LLNL/MuyGPyS#114]

**Comment by bwpriest** [comment]
> Do we still want one of these?

Yes. I'm leaning towards a tutorial. Maybe this can wait on those changes we discussed to simplify the tutorial sample generation chassis. What do you think?

**Comment by bwpriest** [comment]
> Do we still want one of these?

Yes, we will want a basic example in `MuyGPyS.examples`. No need for a tutorial for the time being.

**Comment by esheldon** [comment]
I would also be very interested in a heteroscedastic example.
